examples/datatracks/src/index.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+
+const Prism = require('prismjs');
+const Video = require('twilio-video');
+const getSnippet = require('../../util/getsnippet');
+const getRoomCredentials = require('../../util/getroomcredentials');
+const {sendChatMessage, receiveChatMessages} = require('./helpers');
+
+const P1Connect = document.querySelector('input#p1-connectordisconnect');
+const P2Connect = document.querySelector('input#p2-connectordisconnect');
+const p1ChatLog = document.getElementById('p1-chat-log');
+const p2ChatLog = document.getElementById('p2-chat-log');
+const p1MsgText = document.getElementById('p1-usermsg');
+const p2MsgText = document.getElementById('p2-usermsg');
+const p1Form = document.getElementById('p1-form');
+const p2Form = document.getElementById('p2-form');
+const P1Submit = document.getElementById('P1-msg-submit');
+const P2Submit = document.getElementById('P2-msg-submit');
+
+
+const roomName = "room1"
+let roomP1 = null;
+let roomP2 = null;
+
+/*
+ * Connect to or disconnect the Participant with media from the Room.
+ */
+async function connectToOrDisconnectFromRoom(event, id, room, dataTrack, submitToggle) {
+  event.preventDefault();
+  return room ? disconnectFromRoom(id, room, submitToggle) : await connectToRoom(id, dataTrack, submitToggle);
+}
+
+/**
+ * Connect the Participant with localVideoDiv to the Room.
+ */
+async function connectToRoom(id, dataTrack, submitToggle) {
```

Comment:
The reason why I chose to not use `connectToRoomWithDataTrack` is because I'm connecting both users to the same room via a `roomName`. 

So if I use `connectToRoomWithDataTrack` my two participants join two different rooms.

Reply:
@PikaJoyce You can use `connectToRoomWithDataTrack` for one Participant and get the name of the Room using the `name` property to connect the second Participant.